ci.nix

```diff
@@ -42,9 +40,11 @@
     # We need to use the actual nixpkgs version we're working with here, since the values
     # of 'lib.systems.examples' are not understood between all versions
     let lib = nixpkgs.lib;
-    in lib.optionalAttrs (nixpkgsName == "R2003" && (__elem compiler-nix-name ["ghc865" "ghc884"])) {
+    in lib.optionalAttrs (nixpkgsName == "R2009" && (__elem compiler-nix-name ["ghc865" "ghc884"])) {
     inherit (lib.systems.examples) ghcjs;
-  } // lib.optionalAttrs (system == "x86_64-linux" && (nixpkgsName == "R2009" || (!(__elem compiler-nix-name ["ghc8101" "ghc8102" "ghc810220201118"])))) {
+  } // lib.optionalAttrs (system == "x86_64-linux" && (
+         (nixpkgsName == "R2009" && __elem compiler-nix-name ["ghc8101" "ghc8102" "ghc810220201118"])
+      || (nixpkgsName == "R2003" && __elem compiler-nix-name ["ghc865"]))) {
```

Comment:
I'm not sure this change matches the commit message? This turns off 8.10 + 20.09 and 8.6.5 + 20.03, is that what you meant to do?

Reply:
Ignore me, I can't read.